.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,173 @@
+name: Benchmark
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Specifies the name of the benchmark'
+        required: true
+      ref:
+        description: 'Specifies the ref (e.g. main or a commit sha) to benchmark'
+        default: 'main'
+        required: false
+      cluster:
+        description: 'Specifies which cluster to deploy the benchmark on'
+        default: 'zeebe-cluster'
+        required: false
+      cluster-region:
+        description: 'Specifies the cluster region. Needed to retrieve cluster credentials'
+        default: europe-west1-b
+        required: false
+      load:
+        description: 'Specifies which benchmark components to deploy. `starter`, `timer` and `publisher` can be assigned with the rate at which they publish'
+        default: 'worker, starter=200'
+        required: false
+  workflow_call:
+    inputs:
+      name:
+        description: 'Specifies the name of the benchmark'
+        type: string
+        required: true
```

Comment:
:upside_down_face: Somehow stupid that we have to redeclare them :see_no_evil: but I guess this is the way :shrug:

Reply:
Yeah... :shrug: